assets/js/blocks/cart-checkout/cart/test/block.js

```diff
@@ -32,11 +35,14 @@ describe( 'Testing cart', () => {
 				} }
 			/>
 		);
-		await waitFor( () => {
-			expect(
-				screen.getByText( /Proceed to Checkout/ )
-			).toBeInTheDocument();
-		} );
+		await waitFor( () => expect( fetchMock ).toHaveBeenCalled() );
+		expect(
+			await screen.getByText( /Proceed to Checkout/i )
+		).toBeInTheDocument();
+		/* We have an unfixed bug in our test in which the full cart triggers a POST
+		 * request to `wc/store/cart/update-item` causing the fetch to be called twice.
+		 */
+		expect( fetchMock ).toHaveBeenCalledTimes( 2 );
```

Comment:
As noted in the comment, this is a known issue for now, the block is triggering a post request for `wc/store/cart/update-item` with this body `{key: 1, quantity: 1}`, item with key 1 has a quantity of 2, I'm not sure why is that value is being reduced, but if we change the previewData quantity to 1, this is fixed, if we change the second item quantity to more than 1, it will also trigger a request for it to be reduced, I choose not to change the quantity so that we don't mask this bug.

Reply:
Ya this is odd. It sounds like something specific to the test environment but we should definitely investigate. Could you please make this comment a todo block so that a follow-up issue will automatically be created to handle this (or just create the issue manually if you want)?